scylla-gdb.py

```diff
@@ -4505,8 +4508,11 @@ def invoke(self, arg, from_tty):
             gdb.write('table %s:\n' % schema_ptr(table['_schema']).table_name())
             try:
                 sg_manager = std_unique_ptr(table["_sg_manager"]).get().dereference()
-                for cg in intrusive_list(sg_manager["_compaction_groups"], link='_list_hook'):
-                    scylla_memtables.dump_compaction_group_memtables(cg)
+                for (sg_id, sg_ptr) in absl_container(sg_manager["_storage_groups"]):
+                    sg = seastar_lw_shared_ptr(sg_ptr).get()
```

Comment:
Doesn't this lose backward compatibility?

Reply:
6.0 is already out, so it's indeed better to preserve compatibility.